aten/src/ATen/native/native_functions.yaml

```diff
@@ -3776,15 +3776,6 @@
     CPU: legacy::cpu::_th_potri
     CUDA: legacy::cuda::_th_potri
 
-- func: pstrf(Tensor self, bool upper=True, Scalar tol=-1, *, Tensor(a!) u, Tensor(b!) pivot) -> (Tensor(a!) u, Tensor(b!) pivot)
```

Comment:
I'm worried about C++ API users. I'm not sure if this function is exposed to the c++ API (I hope it's not) but we didn't add a deprecation warning there.

Reply:
@yf225 what are our guarantees on the C++ API? I think we just renamed things like gesv -> solve at some point in the past, but maybe the C++ API was more unstable then?

Reply:
I think `tensor.pstrf` is exposed to the C++ API because things in`Tensor.h` are public APIs.

In general I think it's more important to maintain parity between Python and C++ frontend, and it's okay to break BC for C++ API here. (Here https://pytorch.org/cppdocs/ we also mentioned that "the C++ API should be considered “beta” stability" and "we may make major breaking changes to the backend in order to improve the API".)